Makefile

```diff
@@ -201,8 +201,8 @@ delete-tag:
 	@echo "Deleting tag ${TAG}"
 	@git tag -d ${TAG}
 
-.PHONY: docker-otelcol
-docker-otelcol:
+.PHONY: docker-otelcol-amd64
+docker-otelcol-amd64:
```

Comment:
I think we should leave this as `docker-otelcol` so that we don't have to update all references to this target.

Reply:
resolved